solutions/usaco-860.mdx

```diff
@@ -255,4 +255,108 @@ public class MooyoMooyo {
 
 </JavaSection>
 
+<PySection>
+```python
+import sys
+from collections import deque
+
+sys.stdin = open("mooyomooyo.in", "r")
+sys.stdout = open("mooyomooyo.out", "w")
+
+input = sys.stdin.readline
+
+# possible directions to visit all connected regions
+directions = [
```

Comment:
rename to DIRECTIONS & make it a tuple since it's constant

Reply:
done